light_node/src/configuration.rs

```diff
@@ -74,7 +74,8 @@ pub struct Environment {
     pub storage: Storage,
     pub identity: Identity,
 
-    pub record: bool,
+    pub store_p2p_messages: bool,
```

Comment:
@fafk 
I would put this properties to: configuration.Storage struct
pub store_p2p_messages: bool,
pub store_context_actions: bool,